api/go.mod

```diff
@@ -3,32 +3,29 @@ module github.com/fluxcd/image-automation-controller/api
 go 1.18
 
 require (
-	github.com/fluxcd/pkg/apis/meta v0.14.2
-	github.com/fluxcd/source-controller/api v0.26.0
-	k8s.io/apimachinery v0.24.3
+	github.com/fluxcd/pkg/apis/meta v0.15.0
+	github.com/fluxcd/source-controller/api v0.28.0
+	k8s.io/apimachinery v0.25.0
 	sigs.k8s.io/controller-runtime v0.11.2
 )
 
 // Fix CVE-2022-28948
 replace gopkg.in/yaml.v3 => gopkg.in/yaml.v3 v3.0.0
 
-// Fix CVE-2022-31030
-replace github.com/containerd/containerd => github.com/containerd/containerd v1.6.6
```

Comment:
I wonder why this has been introduced to begin with as containerd shows up nowhere in the deps.